src/python/pants/engine/process.py

```diff
@@ -309,7 +309,7 @@ class InteractiveRunner:
     _scheduler: "SchedulerSession"
 
     def run(self, request: InteractiveProcess) -> InteractiveProcessResult:
-        ExceptionSink.toggle_ignoring_sigint_v2_engine(True)
+        ExceptionSink.toggle_ignoring_sigint(True)
         return self._scheduler.run_local_interactive_process(request)
```

Comment:
It seems like this would be a good place to use a context manager / `with` block... any idea why we stopped doing that?

Reply:
This code never used a `with` block, although I agree that that would be a good improvement, and I might implement that shortly in a followup commit.